common/src/main/java/org/jboss/gm/common/groovy/BaseScript.java

```diff
@@ -134,7 +134,7 @@ public void setValues(Logger logger, InvocationStage stage, File rootDir, Proper
 
         File repoCache;
         try {
-            if (stage == InvocationStage.FIRST || stage == InvocationStage.BOTH) {
+            if (rootProject == null) {
```

Comment:
At first I was going to do something like `if (stage != InvocationStage.LAST)`, but I think this check is even more correct than checking the stage.